src/web/app/components/simple-modal/simple-modal.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="modal-header"
-     [ngClass]="{
+<div class="transparent-overlay" [ngClass]="{ 'display-none': type !== SimpleModalType.LOAD }"></div>
+<div class="modal-header" [ngClass]="{
```

Comment:
Super minor: line break before `ngClass` like the original div tag?

Reply:
Changed! We'll keep the original code formatting since this PR is not specifically for code formatting. 

Just for future reference: I don't think the original formatting for this file is correct (e.g. this line has 5 spaces of indent).